ui-tests/src/test/resources/features/connections/oauth.feature

```diff
@@ -59,7 +59,7 @@ Feature: Connections - OAuth
     Then click on the "Done" button
 
     # finish and save integration
-    When click on the "Save as Draft" button
+    When click on the "Save" button
```

Comment:
can't this step be refactored to use this:

https://github.com/syndesisio/syndesis-qe/blob/04d6aee236532d1fb6d8c2317c069054507d363c/ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/EditorSteps.java#L39

So that you don't have to change it in every test when the label changes?

Reply:
Well yes, but there is a conflict between true BDD and programming best practices :/ I can have 150 steps with "click on X button" and reuse it whenever I want to click on that one button or I can find it dynamically and have just 1 step to find required button based on exact text. So either way there will be many steps or many methods. I prefer this way but if you insist I can refactor it.

Reply:
it depends on you :wink: you can have an enum of the button and find enum based on its name for example

Reply:
Ofc there are ways, but button name from gherkin step should match what user sees anyway. Refactoring of gherkin steps is easy you can even use sed and replace whole step so in case of another change, group replace will be here.

Reply:
¯\\_(ツ)_/¯